crates/ef-testing/src/evm_sequencer/utils.rs

```diff
@@ -47,6 +47,10 @@ pub fn bytes_to_felt_vec(bytes: &Bytes) -> Vec<FieldElement> {
     bytes.to_vec().into_iter().map(FieldElement::from).collect()
 }
 
+pub fn felt_to_bytes(felt: &FieldElement, len: usize) -> Bytes {
+    Bytes::from(&felt.to_bytes_be()[16..len + 16])
```

Comment:
why do we slice from `16`?

Reply:
This comes from the way we implemented bytecode in Kakarot, we only store on 16 bytes. But I see what you mean, I can rename to `high_16_bytes_felt_to_bytes`.